contracts/CourtRoom.sol

```diff
@@ -37,7 +45,6 @@ contract CaseContract is Owned {
 		return clm.claimId;
 	}
```

Comment:
yes. we might do not need this check at all.
Though it might be useful to have a unique id for each claim . 
using "now" as for time stamp for creating the id should can be replaced with an increment   counter of claims. 
If we keep the log via the event mechanism ...we do not need also OpenCases map.